src/partials/hero.html

```diff
@@ -1,14 +1,26 @@
 <section class="hero__section container">
```

Comment:
if you use BAM, you need to add a `hero` block and then use the `hero__section` element.

Reply:
`__section` deleted